content/docs/command-reference/commit.md

```diff
@@ -46,6 +40,20 @@ scenarios are further detailed below.
   reproduce the whole pipeline. If you're sure no pipeline results would change,
   use `dvc commit` to force update the `dvc.lock` or `.dvc` files and cache.
 
+- In cases where we have previously executed a stage (either by writing `dvc.yaml`
+  manually and using `dvc repro`, or with `dvc run`), and later notice that
+  output files or directories created by the stage command, which are already in
+  the workspace, are missing from `dvc.yaml` (`outs` field). We can
+  [add missing outputs to an existing stage](/docs/user-guide/how-to/add-output-to-stage)
+  without having to execute it again. Use `dvc commit` to update the `dvc.lock`
+  file and save outputs to the cache.
+
+- It's always possible to manually execute the terminal command or source code
+  used in a stage without DVC (outputs must be unprotected or removed first in
+  certain cases, see `dvc unprotect`). Once the desired result is reached, use
+  `dvc add` or `dvc commit` as appropriate to update the `dvc.lock` or `.dvc`
+  files and store changed data to the cache.
```

Comment:
So actually this seems wrong. `dvc add` and `.dvc` files don't seem to have anything to do with this case. We're talking about stage commands so only commit and dvc.lock apply here. Please update

p.s. I know this text was there, before but since already we moved it around, let's also fix this up please.

Reply:
Maybe just ``... use `dvc commit` to update the `dvc.lock` file(s) and ...``

Reply:
Done.